app/test_main.py

```diff
@@ -1,3 +1,20 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
```

Comment:
The test cases for (15, 15, [1, 1]) and (23, 23, [1, 1]) seem to be testing the same output. Consider whether both are necessary or if one can be removed to adhere to the 'Don't Repeat Yourself' principle.